source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2726,6 +2724,49 @@ export class Container extends CardElement {
         return element;
     }
 
+    protected truncateOverflow(maxHeight: number): boolean {
+        // Add 1 to account for rounding differences between browsers
+        var boundary = this.renderedElement.offsetTop + maxHeight + 1;
+        this.handleBottomOverflow(boundary);
+        return true;
```

Comment:
Should be true only if at least one element in the container did get truncated, no?

Reply:
With this implementation, Containers will always return true because they can always be truncated to fit any height (in the worst case, all their children get hidden). I'd prefer not to hide Containers as a whole, because then we have to deal with managing inherited visibility properties (tried this earlier, there's some cross-browser differences that are a pain).

Reply:
I'm not getting this. Your own comment reads "Child implementations should return true if the element was truncated to fit within maxHeight, false otherwise." Container always returning true means that the container will appear as always being truncated, when it should only appear as truncated if it truncated (or even fully hid) at least one of its children, no?

Reply:
The key here is that `truncateOverflow` will only get called when there is a need for the container to truncate/hide at least one of its children, so keeping track of whether or not any children were actually hidden or truncated and then returning that value would be functionally equivalent to always returning true.

In any case, I think we can get around having to keep track of whether the element was *actually* truncated -- I'm using the return value from this method more as an indicator of whether the element should be hidden or not. We'll see in the next update.